class-udm-updater.php

```diff
@@ -930,5 +946,54 @@ protected function get_plugin_data($key = '') {
 		}
 		return false;
 	}
+
+	/**
+	 * Remove the use of auto_update index and its value from the *_updater_options option/meta (single/multisite) and change it to auto_update_plugins site option, which is also used by WordPress's core since version 5.5
+	 * This needs to be done in order to maintain auto-updates compatibility between WordPress and UDM and to synchronise the auto-updates setting for both
+	 */
+	public function replace_auto_update_option() {
```

Comment:
@andtown Is there any need for this to be public? I think it should be protected. (Protected is better than private in a library, as it allows developers to extend the class if they wish).

Reply:
@DavidA Nope, It seemed like I was carried away by the making of the filter callback functions that require to be public always. I've replaced the methods' visibility to `protected`.